modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+  <AposContextMenu
+    class="apos-admin-bar_context-button"
+    :menu="menu"
+    :disabled="disabled || (menu.length === 0)"
+    menu-placement="bottom-end"
+    @item-clicked="menuHandler"
+    @open="$emit('menu-open')"
+    @close="$emit('menu-close')"
+    :button="{
+      tooltip: { content: 'More Options', placement: 'bottom' },
+      label: 'More Options',
+      icon: 'dots-vertical-icon',
+      iconOnly: true,
+      type: 'subtle',
+      modifiers: ['small', 'no-motion']
+    }"
+  />
+</template>
+
+<script>
+import { detectDocChange } from 'Modules/@apostrophecms/schema/lib/detectChange';
+import AposPublishMixin from 'Modules/@apostrophecms/ui/mixins/AposPublishMixin';
```

Comment:
For the manuallyPublished computed property. Also likely we will add publish to this menu later.